packages/common/src/Constants.js

```diff
@@ -9,7 +9,8 @@ const interfaceName = {
   FundingRateStore: "FundingRateStore",
   OptimisticOracle: "OptimisticOracle",
   Bridge: "Bridge",
-  GenericHandler: "GenericHandler"
+  GenericHandler: "GenericHandler",
+  MockOracleAncillary: "Oracle"
```

Comment:
@mrice32 I don't think there's any risk to adding this mapping here, which is convenient for the `core/scripts/hardhat/finder.js` script but wanted to highlight it.